src/Routes.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace EMS\CoreBundle;
+
+class Routes
+{
+    public const EditRevision = 'emsco_edit_revision';
+    public const ViewRevision = 'emsco_view_revisions';
+    public const DiscardDraft = 'emsco_discard_draft';
+    public const DraftInProgress = 'emsco_draft_in_progress';
+    public const DraftInProgressAjax = 'emsco_draft_in_progress_ajax';
```

Comment:
const should always by uppercase

```suggestion
    public const EDIT_REVISION = 'emsco_edit_revision';
    public const VIEW_REVISION = 'emsco_view_revisions';
    ...
```